pandas/tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+import numpy as np
+
+from pandas import Series
+import pandas.util.testing as tm
+
+
+@pytest.fixture(params=[tm.makeBoolIndex(10, name='a'),
+                        tm.makeIntIndex(10, name='a'),
+                        tm.makeFloatIndex(10, name='a'),
+                        tm.makeDateIndex(10, name='a'),
+                        tm.makeDateIndex(10, name='a').tz_localize(
+                            tz='US/Eastern'),
+                        tm.makePeriodIndex(10, name='a'),
+                        tm.makeStringIndex(10, name='a'),
+                        tm.makeUnicodeIndex(10, name='a')])
+def indices(request):
+    return request.param
```

Comment:
can you add a doc-string here

Reply:
we prob need to rename these to something like: indices_fixture

Reply:
> indices_fixture

thought I'd follow the previous fixture naming convention like [here](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/indexes/conftest.py#L27)